README.md

```diff
@@ -311,7 +308,7 @@ layout or page's [front matter] YAML in the following form:
 ```yml
 scripts:
   - /path/to/script.js
-  - href: /path/to/script.js
+  - src: /path/to/script.js
```

Comment:
I'm wondering if we should do some explaining on how the setup in the config works in the README?

Reply:
Activating AnchorJS won't require modifying the scripts array, but if you just mean how adding custom scripts works in general I think it's already pretty well covered: https://github.com/18F/uswds-jekyll#scripts

It think adding a blurb about AnchorJS (similar to where we explain Google Analytics and DAP) would be good. Basically just what it is, and how to activate it.